### DISCUSSION TITLE
Flux?

### DISCUSSION THREAD

**Discussion by Sirmadsen** [lllyasviel/stable-diffusion-webui-forge#939] [General]
Flux supporting incoming and if so any timeline?

**Comment by vicento** [comment]
seems that something big happening : [https://github.com/lllyasviel/stable-diffusion-webui-forge/blob/main/backend/diffusion_engine/flux.py](https://github.com/lllyasviel/stable-diffusion-webui-forge/blob/main/backend/diffusion_engine/flux.py)
